contracts/bridges/succinct/SuccinctBridgeReceiver.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.15;
+
+import "../BaseBridgeReceiver.sol";
+import "./ITelepathy.sol";
+
+contract SuccinctBridgeReceiver is BaseBridgeReceiver, ITelepathyHandler {
+    error InvalidSourceChain();
+
+    ITelepathyReceiver public telepathyReceiver;
+    uint32 public sourceChainId;
+
+    constructor(address _telepathyReceiver, uint32 _sourceChainId) {
+        telepathyReceiver = ITelepathyReceiver(_telepathyReceiver);
+        sourceChainId = _sourceChainId;
+    }
+
+    function handleTelepathy(uint32 _sourceChainId, address _sourceAddress, bytes calldata _data) external returns (bytes4)  {
+        if (msg.sender != address(telepathyReceiver)) revert Unauthorized();
```

Comment:
Anyone can execute a message via `telepathyReceiver`, as long as they can have made a call to `TelepathyRouter` on the source chain of the transaction via Telepathy's ZK Light Client.

For Comet, this would be the `govTimelock` submitting a transaction on mainnet, then executing on {Avalanche, BSC, etc.}

More info here: https://docs.telepathy.xyz/telepathy-protocol/overview#receiving-messages